src/python/WMCore/WMBS/MySQL/Create.py

```diff
@@ -65,6 +65,26 @@ def __init__(self, logger = None, dbi = None, params = None):
           """CREATE UNIQUE INDEX uniq_wmbs_file_run_lumi on
              wmbs_file_runlumi_map (fileid, run, lumi)"""
 
+        #
+        # Insert LogCollect site to account for LogCollect output
+        #
+        self.inserts["99_location_logcollect_1"] = \
```

Comment:
I want to talk to @ticoann first to check what this is actually used for before I make further changes.